src/Saxulum/Console/Provider/ConsoleProvider.php

```diff
@@ -32,6 +32,12 @@ public function register(Container $container)
 
         $container['console'] = function () use ($container) {
             $console = new ConsoleApplication($container);
+
+            // automatically detect dispatcher (e.g. in Silex applications)
+            if (!is_null($container['dispatcher'])) {
```

Comment:
thx for the changes, please check for isset($container['dispatcher']), see tests failing